src/main/java/ticTacToe/guiApp/ButtonsCreator.java

```diff
@@ -0,0 +1,32 @@
+package ticTacToe.guiApp;
+
+import javafx.scene.control.Button;
+import ticTacToe.grid.Grid;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ButtonsCreator {
+
+    private final ActionSetter actionSetter;
+    private List<ButtonWrapper> buttonsCreated;
+
+    public ButtonsCreator(ActionSetter actionSetter) {
+        this.actionSetter = actionSetter;
+    }
+
+    public List<ButtonWrapper> createButtons(Grid grid) {
```

Comment:
This method feels a little odd.
We first create the buttons, and add them to a list, then we call another function `setActionsToButtons()` that uses that list (globally, not through an argument...) to add the action to the buttons.

Can we do all of that in one clean go? Setting up buttons feels to "simple" a problem to need multiple loops/passes.